test/data/recordings/TestAccAction_FailedBuild.yaml

```diff
@@ -1,81 +1,182 @@
 ---
-version: 1
+version: 2
```

Comment:
Unfortunately we had to re-record everything because go-vcr 3 uses a different format for the cassettes.